presentation/src/main/java/com/activity_sync/presentation/models/FindUsersResponse.java

```diff
@@ -0,0 +1,124 @@
+package com.activity_sync.presentation.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class FindUsersResponse implements Serializable
```

Comment:
please delete this file, instead edit User model (add there "friends" and "likes" fields).